utils/check-style/check-large-objects.sh

```diff
@@ -9,7 +9,7 @@ TU_EXCLUDES=(
     Aggregator
 )
 
-if find $1 -name '*.o' | xargs wc -c | grep --regexp='\.o$' | sort -rn | awk '{ if ($1 > 50000000) print }' \
+if find $1 -name '*.o' | xargs wc -c | grep --regexp='\.o$' | sort -rn | awk '{ if ($1 > 100000000) print }' \
```

Comment:
What contribytes to big binary size? Is it `joinRightColumnsWithAddtitionalFilter`. I'm afraind we shouldn't just increase the limit because it can lead to build issues, like we had with `RangeHashedDictionary` recently https://github.com/ClickHouse/ClickHouse/pull/61162